chore(add): butternut plugin (#20)
rollup/plugins: packages/butternut/README.md

# rollup-plugin-butternut

> Compress JavaScript with [Butternut](https://github.com/Rich-Harris/butternut).

Reviewer: ```suggestion
🍣 A Rollup plugin which compresses JavaScript with [Butternut](https://github.com/Rich-Harris/butternut).
```

## Installation

> ```bash

Reviewer 2: ```suggestion
```console
```

> npm install --save-dev rollup-plugin-butternut

Reviewer 2: ```suggestion
npm install  @rollup/plugin-butternut --save-dev
```

```

> 

Reviewer 2: ```suggestion
```

## Usage

```js
// rollup.config.js

> import butternut from 'rollup-plugin-butternut';

Reviewer 2: ```suggestion
import butternut from '@rollup/plugin-butternut';
```

> export default {

Reviewer 2: ```suggestion
export default {
  input: 'src/index.js',
  output: {
    dir: 'output',
    format: 'cjs'
  },
  plugins: [butternut()]
};
```

  entry: 'src/main.js',
  dest: 'bundle.js',
  format: 'iife',
  plugins: [ butternut() ]
};
```

> 

Reviewer 2: ```suggestion
## Options

This plugin currently has no available options.

```

> ## License

Reviewer 2: ```suggestion
## Meta

[CONTRIBUTING](/.github/CONTRIBUTING.md)

[LICENSE (MIT)](/LICENSE)
```

> [MIT](LICENSE)

Reviewer 2: ```suggestion
```
